tasks/2016/interactive-bots.md

```diff
@@ -40,59 +39,21 @@ the Task Types below.
 Follow the tutorial below to test run the followup bot on your Zulip
 instance (either local or in your droplet).
 
-* Run the Zulip server and log in the browser as Cordelia.
-
-* Find your user's API key just below the bots section in the settings page.
-This is how Zulip knows the request is from an authorized user.
-
-  You can also create a new bot for a user in settings - it will have its
-  own API key and email. Remember to subscribe it to the streams you want to
-  use the bot in.
-
-* Download your bot's `zuliprc` file, by clicking on the green button, and
-save it as `~/zuliprc-local`.
-
-  Alternatively, you can create this file with credentials:
-
-  ```
-  [api]
-  key=<api-key>
-  email=<email>
-  site=<dev-url>
-  ```
-
-  In the example replace the `<api-key>` with an existing API
-  key and `<email>` with the bot's email. The `<dev-url>` should
-  point to your development environment URL.
-
-  If you're running your bot in the droplet, alongside your Zulip instance,
-  the `<dev-url>` will be `localhost:9991`.
+* Any bot in [`zulip_bots/bots/`](https://github.com/zulip/python-zulip-api/tree/master/zulip_bots/zulip_bots/bots) can [run](http://zulip.readthedocs.io/en/latest/running-bots-guide.html#running-a-bot)
+on a running Zulip server by replacing `<bot-name>` by any of the existing valid bot name
+(under `zulip_bots/bots/`). Replace `<bot-name>` by `followup` for this tutorial.
 
 * Create a `followup` stream on your Zulip server.
 
 * Use `Manage Streams` to subscribe your bot (if you created one) to the
```

Comment:
This step is outdated.

Reply:
We can omit both of the previous steps by just linking to the http://zulip.readthedocs.io/en/latest/running-bots-guide.html#running-a-bot guide.